modules/scaling-hosts-using-the-tuned-profile.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id='scaling-hosts-using-the-tuned-profile-{context}']
+= Scaling hosts using the tuned profile
+
+*Tuned* is a tuning profile delivery mechanism enabled by default in Red Hat
```

Comment:
Is this valid in RHCOS? RHEL isn't supported in 4.0.

If so, s/\*Tuned\*/\`Tuned\` or \_Tuned\_, depending on if it's set off for emphasis.

Reply:
There's a node-tuning-operator which is running tuned

Reply:
Ok! This might need more updates during the review phase.

Reply:
We I think we have docs on the operator specifically, maybe we can link to them here?

Reply:
Maybe - we aren't linking to GitHub repos, so it might be worth reviewing the content and considering including it in the docs before GA.

Reply:
I'm working on the Operator content with Jiri separately. I'll remove this section for now and later link to/incorporate information from that topic if it makes sense to.